ontobio/io/qc.py

```diff
@@ -867,21 +884,16 @@ def test(self, annotation: association.GoAssociation, config: assocparser.AssocP
                 allowed = set([is_active_in])
                 repair_state = RepairState.REPAIRED
         elif namespace == "cellular_component":
-            if term in self.make_protein_complex_descendents_if_not_present(config.ontology):
+            if term == "GO:0032991" or term in self.make_protein_complex_descendents_if_not_present(config.ontology):
                 part_of = association.Curie(namespace="BFO", identity="0000050")
                 if relation not in self.allowed_cc_complex:
-                    if relation in self.repairable_cc_complex:
-                        repaired_annotation = copy.deepcopy(annotation)
-                        repaired_annotation.relation = part_of
-                        repaired_annotation.qualifiers = [part_of]
-                        allowed = self.allowed_cc_complex
-                        repair_state = RepairState.REPAIRED
-                    else:
-                        # Not repairable to part_of, so filter
-                        repaired_annotation = annotation
-                        allowed = self.allowed_cc_complex
-                        repair_state = RepairState.FAILED
-            else:
+                    repaired_annotation = copy.deepcopy(annotation)
+                    repaired_annotation.relation = part_of
+                    repaired_annotation.qualifiers = [part_of]
+                    allowed = self.allowed_cc_complex
+                    repair_state = RepairState.REPAIRED
+                    
+            elif term == "GO:0110165" or term in self.make_cellular_anatomical_entity_descendents_if_not_present(config.ontology) or term == "GO:0044423" or term in self.make_virion_component_descendents_if_not_present(config.ontology):
```

Comment:
With the way the child terms of cellular component are currently set in the ontology:
![image](https://github.com/biolink/ontobio/assets/2678599/42739f9c-0254-4a3d-ab4d-fb7a3250158a)
I think this new `elif` conditional is effectively the same as an `else`. Like, any term that is a CC but not a protein-containing complex can be inferred to be either a cellular anatomical entity or a virion component.

It's nitpicky and doesn't really affect anything so I'm still approvin'!